db-testsuite/src/index.ts

```diff
@@ -7,7 +7,7 @@ import { range } from 'ramda';
 import nanoid from 'nanoid';
 import { DBRouter, DBHandler } from '@binaris/shift-interfaces-koa-server';
 import { Version } from '@binaris/shift-interfaces-koa-server/interfaces';
-import { DB } from '@binaris/shift-db/dist/db';
+import { DB, Q } from '@binaris/shift-db/dist/db';
```

Comment:
Possible to avoid exposing the implementation having `dist/` by adding a `dist.js` which does `module.exports = require('./dist/db.js');` - not sure whether really needed.

Reply:
I think it's safe since it's in the same repo.